config.rb

```diff
@@ -50,6 +50,10 @@ def opsmanual_pages
       .group_by { |page| page.data.section }
   end
 
+  def applications_by_team
+    active_app_pages.group_by(&:team)
```

Comment:
If we make this return an object for the collection, we can move some of the logic in the views to there.